annotations/views.py

```diff
@@ -364,6 +366,62 @@ def collection_texts(request, collectionid):
     return HttpResponse(template.render(context))
 
 
+class RecentActivity(models.Model):
```

Comment:
We created mapping of user and his contributions. So to keep all his contributions at one place we made model class. Also if some more parameters are to be added in future we can add it in the model.